hazelcast-client/src/test/java/com/hazelcast/client/statistics/ClientStatisticsTest.java

```diff
@@ -72,91 +77,89 @@ public void cleanup() {
     }
 
     @Test
-    public void testStatisticsCollectionNonDefaultPeriod()
-            throws Exception {
+    public void testStatisticsCollectionNonDefaultPeriod() {
         HazelcastInstance hazelcastInstance = hazelcastFactory.newHazelcastInstance();
-
         HazelcastClientInstanceImpl client = createHazelcastClient();
+        ClientEngineImpl clientEngine = getClientEngineImpl(hazelcastInstance);
 
         long clientConnectionTime = System.currentTimeMillis();
 
-        ClientEngineImpl clientEngine = getClientEngineImpl(hazelcastInstance);
-
-        // Wait enough time for statistics collection
-        sleepSeconds(statsPeriodSeconds + 1);
+        // wait enough time for statistics collection
+        sleepSeconds(STATS_PERIOD_SECONDS + 1);
         Map<String, String> stats = getStats(client, clientEngine);
 
         String connStat = stats.get("clusterConnectionTimestamp");
-        assertNotNull(connStat);
+        assertNotNull(format("clusterConnectionTimestamp should not be null (%s)", stats), connStat);
         Long connectionTimeStat = Long.valueOf(connStat);
+        assertNotNull(format("connectionTimeStat should not be null (%s)", stats), connStat);
         // time measured by us after client connection should be greater than the connection time reported by the statistics and
         // the difference should not be more than a statistics collection period
-        Assert.assertTrue(clientConnectionTime >= connectionTimeStat
-                && clientConnectionTime - connectionTimeStat < statsPeriodSeconds * 1000);
+        assertTrue(format("connectionTimeStat was %d, clientConnectionTime was %d (%s)",
+                connectionTimeStat, clientConnectionTime, stats),
+                clientConnectionTime >= connectionTimeStat
+                        && clientConnectionTime - connectionTimeStat < STATS_PERIOD_SECONDS * 1000);
 
         String queueSize = stats.get("executionService.userExecutorQueueSize");
-        assertNotNull(queueSize);
+        assertNotNull(format("executionService.userExecutorQueueSize should not be null (%s)", stats), queueSize);
 
-        String mapHits = stats.get("nc." + testMapName + ".hits");
-        assertNull(mapHits);
-        String cacheHits = stats.get("nc." + testCacheName + ".hits");
```

Comment:
Here the test used `"nc." + testCacheName + ".hits"`, which should be `"nc.hz/" + testCacheName + ".hits"` (like in all other lines).